.github/workflows/Lint.yaml

```diff
@@ -30,8 +30,8 @@ jobs:
 
       - name: Run isort
         run: |
-          isort --recursive --check-only openff
+          isort --check-only openff
```

Comment:
```console
$ isort --recursive openff
/Users/mattthompson/mambaforge/envs/qcsubmit-test-basic/lib/python3.9/site-packages/isort/main.py:1274: UserWarning: W0501: The following deprecated CLI flags were used and ignored: --recursive!
  warn(
/Users/mattthompson/mambaforge/envs/qcsubmit-test-basic/lib/python3.9/site-packages/isort/main.py:1278: UserWarning: W0500: Please see the 5.0.0 Upgrade guide: https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html
  warn(
```

https://pycqa.github.io/isort/docs/upgrade_guides/5.0.0.html#-recursive-or-rc

> --recursive or -rc
Prior to version 5.0.0, isort wouldn't automatically traverse directories. The --recursive option was necessary to tell it to do so. In 5.0.0 directories are automatically traversed for all Python files, and as such this option is no longer necessary and should simply be removed.